wgengine/wgengine.go

```diff
@@ -137,4 +137,15 @@ type Engine interface {
 	// Ping is a request to start a discovery ping with the peer handling
 	// the given IP and then call cb with its ping latency & method.
 	Ping(ip netaddr.IP, cb func(*ipnstate.PingResult))
+
+	// RegisterIPPortIdentity registers a given node as temporarily having
```

Comment:
This need to say something about the ownership/mutation rules of the pointer value.

But I think we don't need the whole *Node. Just the userID, right?